decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -629,6 +629,49 @@ def generate_localized_title
     end
   end
 
+  factory :conversation, class: "Decidim::Messaging::Conversation" do
+    transient do
+      skip_injection { false }
+    end
+
+    originator { build(:user) }
+    interlocutors { [build(:user)] }
+    body { Faker::Lorem.sentence }
+    user
+
+    after(:create) do |object|
+      object.participants ||= [originator + interlocutors].flatten
+    end
+
+    initialize_with { Decidim::Messaging::Conversation.start(originator: originator, interlocutors: interlocutors, body: body, user: user) }
+  end
+
+  factory :message, class: "Decidim::Messaging::Message" do
+    transient do
+      skip_injection { false }
+    end
+
+    body { generate_localized_description(:message_body) }
```

Comment:
Should we wait until this is backported or use Faker here and then change it to this method when its available? I'm ok with any of those. Maybe the second option so we can merge this and we don't accumulate work

Reply:
TBH, if i would work towards adding the PR that brings the method. This way we do not forget to patch this, and we do not accidentally break the release branch.

Reply:
Ok lets do that then if that PR is being introduced in the next few hours 😄

Reply:
The methods that i need are part of #12328, which has not been backported yet. Since we need the methods, and we are pending to add them in the nearest future, i have added here so that we can move this PR.